loadtest/control/utils.go

```diff
@@ -13,12 +14,27 @@ import (
 	"time"
 )
 
+func init() {
+	paths := []string{"./testdata/test_text.txt", "./../../testdata/test_text.txt"}
+	var buf []byte
+	var err error
+
+	for _, p := range paths {
+		buf, err = ioutil.ReadFile(p)
```

Comment:
Hmm .. I am slightly confused. This code will keep override the buf variable by reading into it twice. I guess the objective is to use the buf of whichever file appears first ? But then we are continuing if err != nil and also if err == nil. So it will always take the last file irrespective of whatever happens before.

Since this is not a user exposed knob, I would just suggest to directly use "./../../testdata/test_text.txt" and read from it.  The user won't even know that if they create a testdata directory inside control, it would get read before.

Reply:
I assumed that there will be only a single file, in that case it will read only once. If we use  `./../../testdata/test_text.txt` it would work with tests but it won't work with executable therefore I added `./testdata/test_text.txt` because the `test_profile.png` works the same way.

Reply:
Maybe we can just `os.Stat` the file and go ahead with whatever exists.

Reply:
Yes, let's use os.Stat for this. Yes `test_profile.png` also works in the same way, but there is a comment to resolve that to take from the config. I wanted to use that as a config param, but then Claudio just suggested to use it hardcoded.

We need to have an asset folder as a config param for all these testdata things later.